editions/tw5.com/tiddlers/wikitext/HTML Entities.tid

```diff
@@ -0,0 +1,60 @@
+created: 20130204165019000
+modified: 20220822133445149
+tags: WikiText
+title: HTML Entities
+type: text/vnd.tiddlywiki
+
+!! Summary
+
+Use HTML entities to enter characters that cannot easily be typed on an ordinary keyboard. They take the form of an ampersand (`&`), an identifying string, and a terminating semi-colon (`;`), e.g. `&amp;`.
+
+!! Markup
+
+```
+The value of Tiddlers&trade; cannot even be expressed in &pound;, &euro; or &dollar;.
+```
+
+''Displays as:''
+The value of Tiddlers&trade; cannot even be expressed in &pound;, &euro; or &dollar;.
+
+!! Entity References
+
+Comprehensive lists of html entities can be found at...
+
+* [[Mozilla Developer Network -- Entities|https://developer.mozilla.org/en-US/docs/Glossary/Entity]]
+
+* [[HTML Spec official list -- Entities|https://html.spec.whatwg.org/multipage/named-characters.html#named-character-references]]
+
+
+!! Examples Of Common And Useful Entities
+
+|>|>|>|>|>|>| ! HTML Entities |
+| &amp;nbsp; | &nbsp; | no-break space | &nbsp;&nbsp; | &amp;apos; | &apos; | single quote, apostrophe |
+| &amp;ndash; | &ndash; | en dash |~| &amp;quot; | " | quotation mark |
+| &amp;mdash; | &mdash; | em dash |~| &amp;prime; | &prime; | prime; minutes; feet |
+| &amp;hellip; | &hellip; |	horizontal ellipsis |~| &amp;Prime; | &Prime; | double prime; seconds; inches |
+| &amp;copy; | &copy; | Copyright symbol |~| &amp;lsquo; | &lsquo; | left single quote |
+| &amp;reg; | &reg; | Registered symbol |~| &amp;rsquo; | &rsquo; | right  single quote |
+| &amp;trade; | &trade; | Trademark symbol |~| &amp;ldquo; | &ldquo; | left double quote |
+| &amp;dagger; | &dagger; | dagger |~| &amp;rdquo; | &rdquo; | right double quote |
+| &amp;Dagger; | &Dagger; | double dagger |~| &amp;laquo; | &laquo; | left angle quote |
+| &amp;para; | &para; | paragraph sign |~| &amp;raquo; | &raquo; | right angle quote |
+| &amp;sect; | &sect; | section sign |~| &amp;times; | &times; | multiplication symbol |
+| &amp;uarr; | &uarr; | up arrow |~| &amp;darr; | &darr; | down arrow |
+| &amp;larr; | &larr; | left arrow |~| &amp;rarr; | &rarr; | right arrow |
+| &amp;lArr; | &lArr; | double left arrow |~| &amp;rArr; | &rArr; | double right arrow |
+| &amp;harr; | &harr; | left right arrow |~| &amp;hArr; | &hArr; | double left right arrow |
+
+!! Accented Characters
+
+The table below shows how accented characters can be built up by subsituting the underscore (_) into the corresponding character:
```

Comment:
Sadly I don't think that this table is at all clear:

<img width="711" alt="image" src="https://user-images.githubusercontent.com/174761/196912365-1bb59907-0706-4282-9373-3266b1748691.png">

I think you're trying to say that the underscore should be replaced by the letter that you want to add to the accent, but we're not showing enough information to make that clear.

Reply:
I'll add some more examples and rephrase the text

Reply:
Now looks like this: 

![image](https://user-images.githubusercontent.com/374655/197776716-70831534-7485-44b0-90d3-ab9cb952b2c1.png)